modules/tile-converter/src/i3s-converter/helpers/gltf-attributes.ts

```diff
@@ -67,9 +63,14 @@ export function prepareDataForAttributesConversion(
 /**
  * Keep only values for glTF attributes to pass data to worker thread.
  * @param attributes - geometry attributes
+ * @returns attributes with only `value` item
  */
-function getB3DMAttributesWithoutBufferView(attributes: AttributesObject): AttributesObject {
-  const attributesWithoutBufferView = {};
+function getB3DMAttributesWithoutBufferView(attributes: {[k: string]: GLTFAccessorPostprocessed}): {
```

Comment:
Nit: Consider using `Record<string, ...>` instead of `{[k: string], ...}`. On average I think it looks a little cleaner.